Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -4257,6 +4257,12 @@ bool Element::should_indicate_focus() const
     return false;
 }
 
+// https://html.spec.whatwg.org/multipage/interaction.html#tabindex-value
+bool Element::is_focusable() const
+{
+    return HTML::parse_integer(get_attribute_value(HTML::AttributeNames::tabindex)).has_value();
+}
```

Comment:
Commit title: "LibWeb: LibWeb [..]"

Maybe something like this?

"LibWeb: Allow all elements with tabindex attribute to be focusable"